src/anomalib/deploy/inferencers/torch_inferencer.py

```diff
@@ -206,10 +206,15 @@ def pre_process(self, image: np.ndarray) -> torch.Tensor:
         Returns:
             Tensor: pre-processed image.
         """
-        if len(image) == 3:
-            image = image.unsqueeze(0)
+        if len (image.shape) == 3:
+            transposed_image = image.transpose(2,0,1).copy()
+        else:
+            transposed_image = image.copy()
```

Comment:
This is to pass the code quality checks
```suggestion
        transposed_image = image.transpose(2, 0, 1).copy() if len(image.shape) == 3 else image.copy()
```